en/docs/setup/migrating-userstore-managers.md

```diff
@@ -0,0 +1,72 @@
+# Migrating User Stores
+
+Before doing the actual user store migration, it is recommended to do a dry run and analyze the generated report for 
+any recommendations related to the migration. 
+
+## Dry Run
+
+Dry-run capability in the user store migrator will allow the migration utility to validate the system for the 
+user store configurations and generate a report regarding the migration compatibility. In the generated migration report, 
+if there are any warnings it is recommended to contact the WSO2 support to identify the best migration strategy.
+
+### How to Run
+
+Configure the migration report path using the value of “reportPath” in the migration-config.yaml. 
```

Comment:
...using the value of `reportPath` in `<IS_HOME>/migration-resources/migration-config.yaml`.